osmoutils/accum/accum_helpers.go

```diff
@@ -35,8 +39,24 @@ func getPosition(accum AccumulatorObject, name string) (Record, error) {
 func getTotalRewards(accum AccumulatorObject, position Record) sdk.DecCoins {
 	totalRewards := position.UnclaimedRewards
 
+	// TODO: add a check that accum.value is greater than position.InitAccumValue
```

Comment:
This is non-blocking for fees. I don't want to get too wrapped up in the details to block further fee work.

We have a bunch of TODOs in this package since we still are iterating and prototyping the final design. I think it would make sense to come back and address all the details once we finalize the `accum` package and finish its integration into fees and incentives.

Let me know if that  doesn't convince you to keep this as todo for now